sites/main-site/src/content/blog/2024/seqera-containers-part-2.mdx

```diff
@@ -0,0 +1,646 @@
+| `conda_arm`            | <span class="badge text-bg-success">New</span>         | Conda lockfiles for `linux/arm64`                                                  |
+| `conda_env`            | <span class="badge text-bg-success">New</span>         | Conda environments with local environment resolution                               |
+| `mamba`                | <span class="badge text-bg-warning">Updated</span>     | Conda lockfiles for `linux/amd64`, using Mamba                                     |
+| `mamba_arm`            | <span class="badge text-bg-success">New</span>         | Conda lockfiles for `linux/arm64`, using Mamba                                     |
+| `mamba_env`            | <span class="badge text-bg-success">New</span>         | Conda environments with local environment resolution, using Mamba                  |
+
+:::tip{.fa-comment-question title="Singularity: oras or https?" collapse}
+
+Unfamiliar with `oras://`? Don't worry, it's relatively new in the field.
+It's a new protocol to reference container images, similar to `docker://` or `shub://`.
+It allows Singularity to interact with any OCI ([Open Container Initiative](https://opencontainers.org/))
+compliant registry to pull images.
+
+Using `oras` has some advantages:
+
+-   Singularity handles pulls in the process task, rather than it happening in the Nextflow head job
+    -   This means less resource usage on the head node, and more parallelisation
+-   Singularity can use authentication to pull from private registries
+    (see [Singularity docs](https://docs.sylabs.io/guides/main/user-guide/cli/singularity_registry.html) for more information).
+
+However, there are some downsides:
+
+-   Shared cache Nextflow options such as `$NXF_SINGULARITY_CACHEDIR` and `$NXF_SINGULARITY_LIBRARYDIR` do not work
+-   Singularity must be installed to download images for offline use
+-   `oras` is not supported by all versions of Singularity / Apptainer
+
+As such, we will continue to use `https` downloads for Singularity `SIF` images for now.
+However, we will start to provide new `-profile singularity_oras` profiles for anyone who
+would prefer to fetch images using the newer `oras` protocol.
+
+If you'd like to know more, check out the amazing [bytesize talk](https://nf-co.re/events/2024/bytesize_singularity_containers_hpc)
+by Marco Claudio De La Pierre ([@marcodelapierre](https://github.com/marcodelapierre/)) from June 2024:
+
+<YouTube id="https://www.youtube.com/watch?v=zoCC_dkhjD0" poster="http://i3.ytimg.com/vi/zoCC_dkhjD0/hqdefault.jpg" />
+
+:::
+
+## Modules
+
+All nf-core pipelines use a single container per process, and the majority of processes are
+encapsulated within shared modules in the [nf-core/modules](https://github.com/nf-core/modules) repository.
+As such, we must start with containers at the module level.
+
+:::tip{.fa-brands.fa-github title="GitHub issue"}
```

Comment:
```suggestion
As such, we must start with containers at the module level.

:::tip{.fa-brands.fa-github title="GitHub issue"}

The changes made to the module files that are as follows: 
```

Because this wasn't exactly clear what we are about to describing here, and the inline code blocks in headers also are a bit iffy

Reply:
I don't understand this suggested change sorry..?

Reply:
Add the line `The changes made to the module files that are as follows: ` to explain that in the following sections we are describing what changes to specific files we will have to make.

The paragraph above the tip just says:

>  All nf-core pipelines use a single container per process, and the majority of processes are
encapsulated within shared modules in the [nf-core/modules](https://github.com/nf-core/modules) repository.
As such, we must start with containers at the module level.

Start with doing 'what'? exactly

I guess it was assumed the header names of the two following sections (`main.nf`, `meta.yaml`) would be self explantory, but as the headers are in-line code blocks it wasn't clear to me it was a new section.

Reply:
How about adding to the headings instead?

### Changes to `main.nf`

### Changes to `meta.yml`

Reply:
Done in e32935e8f